generate.py

```diff
@@ -246,7 +246,7 @@ def to_json(self):
         id_="cpp",
         name="C++",
         keys=["cpp", "c++"],
-        exts=[".cc", ".cpp", ".h"],
+        exts=[".cc", ".cpp", ".cxx", ".c++", ".pcc", ".tpp", ".h", ".hh", ".hpp", ".hxx", ".inl", ".ipp"],
```

Comment:
maybe you could add a comment here saying this list was derived from tokei source with a URL link